test/detect/vbnet/default.txt

```diff
@@ -1,5 +1,5 @@
-Import System
-Import System.IO
+Imports System
+Imports System.IO
```

Comment:
Is this just new syntax?

Reply:
I don't know about new, but I can only find `imports` in the [documentation](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/statements/imports-statement-net-namespace-and-type) and the list of keywords:
https://github.com/highlightjs/highlight.js/blob/6698b8a5959c398568c7c437fe028a6e3167c776/src/languages/vbnet.js#L18